### ISSUE TITLE
[$500] [Simplified Collect][Distance Rates] Implement CreateDistanceRatePage

### ISSUE THREAD

**Issue by luacmartins** [Expensify/App#37512]
Implement [PolicyNewDistanceRatePage](https://docs.google.com/document/d/1hJlRWUQlOAcDsq23zWWITmC83tuDs8rfI4rPAwpjhc4/edit#bookmark=id.qf7ilrr5wfzd) following the specifications in the design doc

cc @MrMuzyk
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01350a6fa3ed44b2b0</li>
        <li>Upwork Job ID: 1763222767227588608</li>
        <li>Last Price Increase: 2024-02-29</li>
    </ul>
</details>

**Comment by MrMuzyk** [comment]
I am Michał from Callstack - expert contributor group. I’d like to work on this job.

**Comment by mallenexpensify** [comment]
Since @MrMuzyk is assigned, I believe this should be weekly.

**Comment by situchan** [comment]
This is new feature. I think regression test will be added after simplified collect is done. Not needed for this specific issue.

**Comment by luacmartins** [comment]
Bump @mallenexpensify for payment

**Comment by mallenexpensify** [comment]
@situchan , can you please accept the job and reply here once you have? 
https://www.upwork.com/jobs/~01191015db3d9f269d

**Comment by luacmartins** [comment]
Still waiting on payment

**Comment by mallenexpensify** [comment]
@situchan is OOO, bumping to weekly.

**Comment by luacmartins** [comment]
Pending payment

**Comment by mallenexpensify** [comment]
@situchan  back soon

**Comment by mallenexpensify** [comment]
Still waiting for @situchan to return.

**Comment by mallenexpensify** [comment]
@situchan can you please accept the job and reply here once you have? 
https://www.upwork.com/jobs/~01808847a707e339cb

**Comment by situchan** [comment]
Accepted thanks

**Comment by mallenexpensify** [comment]
Contributor+: @situchan paid $500 via Upwork. 

@situchan @luacmartins I'm assuming we want a TestRail GH for this since it's a new feature. If you agree, @situchan , please provide the steps to add.  Thx

**Comment by mallenexpensify** [comment]
Ping on needing a TestRail case for this

**Comment by situchan** [comment]
### Regression Test Proposal
1. Login account with paid policy that has access to distance rates page
2. Go to Account settings > Workspaces > paid policy > Distance rates
3. Click "Add rate" and enter amount
4. Click "Save
5. Verify that new rate is added successfully

**Comment by luacmartins** [comment]
No TR tests needed. We already added this as part of the project wrap up. Gonna close this issue.
